src/renderer/components/DeviceAction/actions/app.js

```diff
@@ -195,6 +195,10 @@ const useHook = (device: ?Device, appRequest: AppRequest): AppState => {
             // no debounce for allow event
             return empty();
           }
+          if (!s.device) {
+            // extra debounce for the disconnect event
+            return interval(2000);
+          }
           // default debounce (to be tweak)
           return interval(1500);
```

Comment:
just change the 1500 to 2000 I think ?

Reply:
To me all this debouncing is a bit hacky to begin with i dont get why we receive a device disconnection after opening an app in the first place.

generalizing the debounce just worsen responsiveness on all other actions but we can set it at 2s all the time if you want.

Reply:
I agree that the hardware is hacky. it disconnect and reconnect when you enter an app. so yeah we have no other choices but to debouce this to not have the user seeing it.

Reply:
just set it by default to 2s :+1: